### PR TITLE
Ft login to phantom

### PR DESCRIPTION
**Pull request by NyiingangoPacifique**
#### What does this PR do?

- Login to Phantom App

#### Description of Task to be completed?
- User must be able to login to phantom
- Appropriate error messages are displayed on unsuccessful attempts
- Appropriate error messages are displayed on invalid inputs (Form validation).
#### How should this be manually tested?
-first pull branch `git pull ft-login-to-phantom`
-install dependencies `npm install`
-then run` npm run dev`
#### Screenshots
Screenshot from 2022-04-12 19-00-13

### Responsive
Screenshot from 2022-04-12 19-01-06

### Displaying error from server
Screenshot from 2022-04-12 19-02-43

### CODE REVIEW

File: src/components/Navbarother.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
```
Comment (JPaul23) [inline src/components/Navbarother.jsx:1]: Which navbar should we use? there is two navbar components

Comment (christianR15c) [reply src/components/Navbarother.jsx:1]: there is a navbar for the landing page and another one for the remaining pages you should use the one @NyiingangoPacifique is using but I think he need to make some changes on it

Comment (NyiingangoPacifique) [reply src/components/Navbarother.jsx:1]: what change exactly

File: src/state/actions/authAction.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
```
Comment (SimonImanigirimpuhwe) [inline src/state/actions/authAction.js:1]: You're using `axios`, yet you don't have this package installed

File: src/components/auth/Login.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+          this.validatePassword(e.target.value);
+         }
+       if(this.state.emailError===false && this.state.passwordError===false){
+          this.setState({
+            isDisabled:false
+          })
+       }
+      }
+
+    // onChange = e => {
```
Comment (SimonImanigirimpuhwe) [inline src/components/auth/Login.js:91]: Remove commented lines

File: src/components/auth/Login.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+        e.preventDefault();
+        console.log("hello");
+       const { email, password } = this.state;
+
+       const user = {
+           email,
+           password
+       };
+       console.log(user.email);
+       console.log(user.password);
```
Comment (SimonImanigirimpuhwe) [inline src/components/auth/Login.js:105]: Remove `console.log`s

File: src/state/actions/actionTypes.js [hunk 5]
```diff
@@ -1,3 +1,8 @@
 export const INCREMENT = "INCREMENT";
 export const DECREMENT = "DECREMENT";
-export const RESET = "RESET";
\ No newline at end of file
+export const RESET = "RESET";
```
Comment (SimonImanigirimpuhwe) [inline src/state/actions/actionTypes.js]: Remove counter codes

File: src/state/actions/authAction.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+export const login =({ email,password }) => dispatch => {
+
+ //request body
+
+ const body = JSON.stringify({ email,password })
+ console.log(email);
+ console.log(body);
+  axios
+    .post('https://phantom-api-pirates.herokuapp.com/users/login', {
+      email : email,
```
Comment (SimonImanigirimpuhwe) [inline src/state/actions/authAction.js:20]: Make this `URL` as an environment variable

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Please check if all packages are installed

File: src/components/auth/Login.js (hunk 3 above)
Comment (NyiingangoPacifique) [reply src/components/auth/Login.js:91]: yeah

File: src/components/auth/Login.js (hunk 4 above)
Comment (NyiingangoPacifique) [reply src/components/auth/Login.js:105]: it was for testing purpose let's me remove them

File: src/state/actions/actionTypes.js (hunk 5 above)
Comment (NyiingangoPacifique) [reply src/state/actions/actionTypes.js]: okay

File: src/state/actions/authAction.js (hunk 6 above)
Comment (NyiingangoPacifique) [reply src/state/actions/authAction.js:20]: yeah sure

**Pull request closed without merging**